### PR TITLE
OpenEXR split & Denoiser Tonemapper

### PR DESCRIPTION
**Pull request by AnastaZIuk**
status:

- OpenEXR split is **done** and **tested**
- Denoiser Tonemapper CommandLineParser is **done** and **tested**
- Denoiser Tonemapper example needs to be **completed** - @devshgraphicsprogramming it's your turn

### CODE REVIEW

**Comment by devshgraphicsprogramming** [comment]
address the comment by @Crisspl and add the camera transform parameter

**Comment by AnastaZIuk** [comment]
okay, @devshgraphicsprogramming you can take control

File: examples_tests/39.DenoiserTonemapper/CommandLineHandler.hpp [hunk 1]
```diff
@@ -5,128 +5,193 @@
...
-	public:
-
-		CommandLineHandler(const int argc, core::vector<std::string> argv, IAssetManager* am);
-
-		auto getVariables() { return &rawVariables; }
-		auto getStatus() { return status; }
-		auto getMode() { return mode; }
-
-	private:
+	namespace asset
```
Comment (devshgraphicsprogramming) [inline examples_tests/39.DenoiserTonemapper/CommandLineHandler.hpp]: don't add any code to the `irr::asset` namespace that is not in engine core

File: examples_tests/39.DenoiserTonemapper/exampleInputArguments.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 -OPENEXR_FILE=in.exr
```
Comment (devshgraphicsprogramming) [inline examples_tests/39.DenoiserTonemapper/exampleInputArguments.txt]: it was supposed to be one line with no newline with all arguments per file!

**Comment by devshgraphicsprogramming** [comment]
See latest review and I will

**Comment by devshgraphicsprogramming** [comment]
close this PR

**Comment by AnastaZIuk** [comment]
closing and moving to https://github.com/buildaworldnet/IrrlichtBAW/pull/439

**Pull request closed without merging**